src/axom/slic/core/LogStream.hpp

```diff
@@ -120,6 +120,32 @@ class LogStream
    */
   virtual void push() {};
 
+  /*!
```

Comment:
Should these methods be exposed in the public interface? Specifically, are there concerns about users calling them?